packages/datetime/src/dateRangeSelectionStrategy.ts

```diff
@@ -22,19 +22,22 @@ export interface IDateRangeSelectionState {
 export class DateRangeSelectionStrategy {
     public static getNextState(currentRange: DateRange,
                                day: Date,
-                               boundaryToModify?: DateRangeBoundary,
-                               allowSingleDayRange?: boolean) {
+                               allowSingleDayRange: boolean,
+                               boundaryToModify?: DateRangeBoundary): IDateRangeSelectionState {
         if (boundaryToModify != null) {
-            return this.getNextStateForBoundary(currentRange, day, boundaryToModify, allowSingleDayRange);
+            return this.getNextStateForBoundary(currentRange, day, allowSingleDayRange, boundaryToModify);
         } else {
             return this.getDefaultNextState(currentRange, day, allowSingleDayRange);
         }
     }
 
     private static getNextStateForBoundary(currentRange: DateRange,
                                            day: Date,
-                                           boundaryToModify: DateRangeBoundary,
-                                           allowSingleDayRange: boolean) {
+                                           allowSingleDayRange: boolean,
+                                           boundaryToModify: DateRangeBoundary) {
+        // `boundaryToModify` is the clearer name for the exposed parameter, but `boundary` is a
```

Comment:
i really have no problem just called the parameter `boundary` and being done with it. better place for this comment is in `/** */` on the function itself to explain usage.